### ISSUE TITLE
Make cargo return shipments complaints easier to read.

### ISSUE THREAD

**Issue by williamcll** [BeeStation/NSV13#1928]
## Reproduction:
![image](https://user-images.githubusercontent.com/43061493/171106592-3568f010-11e0-461a-aff8-e1456d834d60.png)
One problem with cargo missions right now is that it is hard to figure out why stations are returning and reporting wrong deliveries. Players are absolutely they have collected the right materials for delivery yet stations still complains about the cargo being wrong. This goes back and forth for hours to either someone figures it out of armada kills the ship.

**Comment by jzburda** [comment]
Cargo shipments definitely need to be easier to read. But that isn't the issue. We tried shipping a tech disk both packed and unpacked for a directive, both came back "That wasn't what we were asking for! Here's your shit back, try again!"

**Comment by bluewasabi24** [comment]
It's mostly the mission that's "food+booze but also please wrap it a specific way" that's causing the most problems. The parameters of this mission have changed so many times, and it's at the point where the wording doesn't make any goddamn sense. We've tried every combination of wrapped and unwrapped and it's still not working.

**Comment by Bokkiewokkie** [comment]
> I could speculate that this one was because someone unwrapped the original packaging and that caused the regex (is there even regex?!?!?!) to fail
Yeah if you remove the original packaging it'll automatically fail the mission right now
